site/content/docs/1.25/configuration.md

```diff
@@ -286,7 +286,7 @@ data:
     # kubeconfig: /path/to/.kube/config
     #
     # Disable RFC-compliant behavior to strip "Content-Length" header if
-    # "Tranfer-Encoding: chunked" is also set.
+    # "Transfer-Encoding: chunked" is also set.
```

Comment:
I probably should comment on this once...
* There's one [GitHub issue](https://api.github.com/repos/ReactiveX/RxNetty/issues/470) with this typo, but the discussion uses the correct spelling.
* There's a single [Oracle KB entry](https://support.oracle.com/knowledge/Middleware/2778217_1.html) with the typo in its title where the body similarly uses the correct spelling.
* There's _currently_ a typo in a comment in https://www.w3.org/Tools/Mail-Transcode/mail-transcode.c (but I just reported that to its author) and it references the RFC which spells it correctly (and it isn't even the same header, it's just a similar one...).

But, overall this typo isn't used in the wild in actual network operations, it's just a commonly made typo (so common that searching it gives me two rounds of `Did you mean: "Transfer-Encoding"`)